packages/flutter_tools/lib/src/ios/xcode_debug.dart

```diff
@@ -310,6 +317,7 @@ class XcodeDebug {
           _xcode.xcodeAppPath,
           '-g', // Do not bring the application to the foreground.
           '-j', // Launches the app hidden.
+          '-F', // Open "fresh", without resorting windows.
```

Comment:
This is not actually related to the fix, but just a same improvement. I noticed sometimes when opening Xcode, it will open the project as well as other previously opened Xcode windows. This prevents that and just opens the targeted project.